cincoctrl/cincoctrl/findingaids/filters.py

```diff
@@ -9,7 +9,7 @@ class Meta:
         model = FindingAid
         fields = {
             "repository": ["exact"],
-            "collection_title": ["contains"],
+            "collection_title": ["icontains"],
```

Comment:
Makes me wonder if at some point we should consider adding indices for whatever we are filtering on?